src/App.js

```diff
@@ -1,27 +1,19 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/no-unescaped-entities */
+import Calculator from './components/Calculator';
+import logo from './media/logo.png';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="app">
+      <img
+        src={logo}
+        alt="Logo"
+        className="logo"
+      />
+      <div className="app-container">
+        <h2 className="app-header">Let's do some math!</h2>
+        <Calculator />
```

Comment:
**Changes Required**
Good effort with the function component, however;
- [ ] The `App Component` should be a `Class Component` as specified in the project requirement. Let us refactor it into a class component.

Reply:
> Issue resolved .. thank you for your time and feedback